File: pbwebwcl/src/app/validazione/commons/dto/esito-operazioni.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

export class EsitoOperazioni {
    constructor(
        public esito: boolean,
        public msg: string,
        public params: Array<string>
    ) { }
}

File: pbservizit/pbservizit-jar/src/main/java/it/csi/pbandi/pbservizit/dto/EsitoOperazioni.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbservizit.dto;

import java.io.Serializable;

public class EsitoOperazioni implements Serializable {

	static final long serialVersionUID = 1;

	private Boolean esito = null;
	private String msg = null;
	private String[] params = null;
	
	public void setEsito(Boolean val) {
		esito = val;
	}

	public Boolean getEsito() {
		return esito;
	}

	public void setMsg(String val) {
		msg = val;
	}

	public String getMsg() {
		return msg;
	}

	public void setParams(String[] val) {
		params = val;
	}

	public String[] getParams() {
		return params;
	}

}
